pkg/transforms/common.go

```diff
@@ -811,7 +815,8 @@ func commonStatusConditions(kind string, group string, r *unstructured.Unstructu
 	}
 	if !config.Cfg.CollectStatusConditions {
 		switch kind + group {
-		case "VirtualMachine.kubevirt.io", "Node", "Pod", "Search.search.open-cluster-management.io", "MultiClusterHub.operator.open-cluster-management.io":
+		case "VirtualMachine.kubevirt.io", "Node", "Pod", "Search.search.open-cluster-management.io",
+			"MultiClusterHub.operator.open-cluster-management.io", "NetworkAddonsConfig.networkaddonsoperator.network.kubevirt.io":
```

Comment:
Is it easy to add this to the configuration? I'm thinking it would be similar to how you added `extractAnnotations`.